dex-it/src/test/scala/com/wavesplatform/it/sync/api/GetSettingsUtilsTestSuite.scala

```diff
@@ -0,0 +1,26 @@
+package com.wavesplatform.it.sync.api
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.it.MatcherSuiteBase
+
+class GetSettingsUtilsTestSuite extends MatcherSuiteBase {
+
+  override protected val dexInitialSuiteConfig: Config =
+    ConfigFactory.parseString(
+      s"""waves.dex {
+         |  seed = "test"
+         |  price-assets = [ "$UsdId", "$BtcId", "WAVES" ]
+         |}
+       """.stripMargin
+    )
+
+  override protected def beforeAll(): Unit = {
+    wavesNode1.start()
+    broadcastAndAwait(IssueUsdTx, IssueBtcTx)
+    dex1.start()
+  }
+
+  "GET matcher/debug/config should return config without restricted key-parts (e.g. seed)" in {
+    dex1.api.config.toString should not contain "seed"
```

Comment:
Does `toString` prints the content of `Config` and not something like `Config@4b32032` ?